gluster/file/cmd/glusterfile-provisioner/glusterfile-provisioner.go

```diff
@@ -352,15 +352,13 @@ func getVolumeID(pv *v1.PersistentVolume, volumeName string) (string, error) {
 
 func (p *glusterfileProvisioner) Delete(volume *v1.PersistentVolume) error {
 
-	glog.V(1).Infof("deleting volume")
+	glog.V(1).Infof("deleting volume %s", volume.Spec.Glusterfs.Path)
```

Comment:
this is not quite accurate, how about "deleting volume, path %s"?

Reply:
@rootfs done.